roles/openshift_master/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
 - name: restart master
   service: name={{ openshift.common.service_type }}-master state=restarted
-  when: not openshift_master_ha | bool
+  when: not (openshift_master_ha | bool or skip_master_restart | default(false))
```

Comment:
Why is `default(false)` needed here?  Is there a case where one of these variables isn't set?